allennlp/data/fields/tensor_field.py

```diff
@@ -76,3 +77,15 @@ def __eq__(self, other) -> bool:
     def array(self):
         """This is a compatibility method that returns the underlying tensor as a numpy array."""
         return self.tensor.numpy()
+
+    @overrides
+    def human_readable_repr(self) -> JsonDict:
+        shape = list(self.tensor.shape)
+        std = torch.std(self.tensor.float()).item()
+        ave = torch.mean(self.tensor.float()).item()
+        return {
+            "shape": shape,
+            "element_std": std,
+            "element_ave": ave,
```

Comment:
As a former statistician, I prefer to keep the "mean" terminology:

```suggestion
        mean = torch.mean(self.tensor.float()).item()
        return {
            "shape": shape,
            "element_std": std,
            "element_mean": mean,
```

Reply:
Sounds good! Follow the expert :D